src/transactions/types.ts

```diff
@@ -48,3 +51,15 @@ export type SignTransactionOptions = RequestOptions<
   SignTransactionPayload,
   SignTransactionResponse
 >;
+
+export interface SignMultipleTransactionsPayload extends RequestPayload {
+  message: string;
+  psbts: PsbtPayload[];
```

Comment:
Just checking on this, do we expect some psbts to have broadcast set to true and some to false or should boroadcast maybe be pulled out of the PsbtPayload type?

Reply:
yeah this was the point i brought up during the daily before maybe @yknl can chime in on that one

Reply:
also only passing an address without specifying and index

Reply:
We should pull it out. Having some transactions broadcasted and some not would be bad UX. Developers can always make another request for transactions that they don't intend to broadcast immediately.

Reply:
@m-aboelenein will you change the interface here so that `broadcast` boolean property is related to all of the psbts sent to sign?